pkg/awsutils/awsutils.go

```diff
@@ -1305,12 +1311,13 @@ func (cache *EC2InstanceMetadataCache) AllocIPAddresses(eniID string, numIPs int
 	output, err := cache.ec2SVC.AssignPrivateIpAddressesWithContext(context.Background(), input, userAgent)
 	awsAPILatency.WithLabelValues("AssignPrivateIpAddresses", fmt.Sprint(err != nil)).Observe(msSince(start))
 	if err != nil {
-		log.Errorf("Failed to allocate a private IP addresses on ENI %v: %v", eniID, err)
-		awsAPIErrInc("AssignPrivateIpAddresses", err)
 		if containsPrivateIPAddressLimitExceededError(err) {
-			log.Debug("AssignPrivateIpAddresses returned PrivateIpAddressLimitExceeded")
+			log.Debug("AssignPrivateIpAddresses returned PrivateIpAddressLimitExceeded, probably because the data store was out of sync." +
```

Comment:
This error is seen by a ec2 call and when the Cx runs out of Ip's right? Should we change this to something like "Check IP usage limits" ?

Reply:
This doesn't happen when the customer runs out of IPs, but when we call `AssignPrivateIpAddresses` asking for more secondary IPs than the ENI can support. I'll reword this log message. Also, after this change we should not store "empty" ENIs in the datastore.